### PR TITLE
SWC-5736: show the Team Actions tools menu if any menu items are set …

### PR DESCRIPTION
**Pull request by jay-hodgson**
…to visible

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java [hunk 1]
```diff
@@ -159,6 +159,7 @@ public void clear() {
 		deleteTeamItem.setVisible(false);
 		leaveTeamItem.setVisible(false);
 		publicJoinField.setVisible(false);
+		toolsMenu.setVisible(false);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java:162]: During view reset, hide the Team Actions menu button (so by default this is not visible).

File: src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java [hunk 2]
```diff
@@ -189,13 +190,21 @@ public void setPresenter(Presenter presenter) {
 	@Override
 	public void showMemberMenuItems() {
 		leaveTeamItem.setVisible(true);
+		// TODO: remove next line to take out of alpha mode
+		teamProjectsItem.setVisible(DisplayUtils.isInTestWebsite(cookieProvider));
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java:194]: Must be a team member for this command to work, so only show it when other member items are shown.

File: src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java [hunk 3]
```diff
@@ -189,13 +190,21 @@ public void setPresenter(Presenter presenter) {
...
+		toolsMenu.setVisible(true);
 	}
 
 	@Override
 	public void showAdminMenuItems() {
 		deleteTeamItem.setVisible(true);
 		editTeamItem.setVisible(true);
 		inviteMemberItem.setVisible(true);
+		// TODO: remove next line to take out of alpha mode
+		teamProjectsItem.setVisible(DisplayUtils.isInTestWebsite(cookieProvider));
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java:205]: Or if the user is an admin of the team.

File: src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java [hunk 4]
```diff
@@ -189,13 +190,21 @@ public void setPresenter(Presenter presenter) {
 	@Override
 	public void showMemberMenuItems() {
 		leaveTeamItem.setVisible(true);
+		// TODO: remove next line to take out of alpha mode
+		teamProjectsItem.setVisible(DisplayUtils.isInTestWebsite(cookieProvider));
+		
+		toolsMenu.setVisible(true);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java:196]: If any menu items are shown, show the Team Actions button.

**Review by xschildw — APPROVED** [review]

**Pull request merged**